fr_FR/docs/01-writing-tests.md

```diff
@@ -146,7 +146,7 @@ test.todo('il faudra penser à écrire cela plus tard');
 Vous pouvez signaler que vous devez écrire un test en série :
 
 ```js
-test.serial.todo('penser à l''écrire plus tard');
+test.serial.todo("penser à l'écrire plus tard");
```

Comment:
You're right, but all AVA examples use the simple quote, so to maintain consistency in the examples, it would be better to do this:

```javascript
test.serial.todo('penser à l\'écrire plus tard');
```

Can you change in this way?

Reply:
FIxed ✔️